execution/execution_loop.py

```diff
@@ -62,6 +61,8 @@ def execute(args, exec_name, model_name, model, optim_func, input_func, grad_fun
         finally:
             import torchdynamo
             from torchdynamo.optimizations.training import aot_autograd_speedup_strategy
+            if 'GNN' in model_name:
```

Comment:
Could you pull this out and put it under a new section where you try to import `torchgeometric` only if it is in the benchmark and install the `torchgeometric` so the user doesn't have to figure out how to install it?